tests/std/tests/P2442R1_views_chunk/test.cpp

```diff
@@ -0,0 +1,603 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <forward_list>
+#include <ranges>
+#include <span>
+#include <type_traits>
+#include <vector>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+
+template <class Rng>
+concept CanViewChunk = requires(Rng&& r) {
+    views::chunk(forward<Rng>(r), 2);
+};
+
+constexpr auto equal_ranges = [](auto&& left, auto&& right) { return ranges::equal(left, right); };
+
+template <ranges::input_range Rng, class Expected>
+constexpr bool test_one(Rng&& rng, Expected&& expected) {
+    using ranges::bidirectional_range, ranges::common_range, ranges::forward_range, ranges::random_access_range,
+        ranges::sized_range;
+    using ranges::chunk_view, ranges::begin, ranges::end, ranges::equal, ranges::iterator_t, ranges::sentinel_t,
+        ranges::prev;
+    constexpr bool is_view = ranges::view<remove_cvref_t<Rng>>;
+
+    using V = views::all_t<Rng>;
+    using R = chunk_view<V>;
+
+    STATIC_ASSERT(ranges::view<R>);
+    STATIC_ASSERT(forward_range<R> == forward_range<Rng>);
+    STATIC_ASSERT(bidirectional_range<R> == bidirectional_range<Rng>);
+    STATIC_ASSERT(random_access_range<R> == random_access_range<Rng>);
+    // Validate borrowed_range
+    static_assert(ranges::borrowed_range<R> == (ranges::borrowed_range<V> && forward_range<V>) );

```

Comment:
Is there a reason that this is a `static_assert` instead of a `STATIC_ASSERT`?

Reply:
just good ol' sloppiness